conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('openbsd'):
```

Comment:
@groutr, I'd like for @NewbiZ to do my suggested change here before merging, so this is not related only to `openbsd` but to all BSDs.